var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -57,5 +63,12 @@ class Mesa(AutotoolsPackage):
     depends_on('presentproto@1.0:', type='build')
     depends_on('pkg-config@0.9.0:', type='build')
 
+
+    def configure_args(self):
+        make('autoreconf -fi')
```

Comment:
i think `configure_args` is intended only to return flags, not to run anything in `AutotoolsPackage` package. Maybe @alalazo can comment on where this line should go.